src/styles/blocks/card.scss

```
@@ -0,0 +1,94 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  color: $main-text-color;
+
+  &__image {
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+    background-size: 100% 100%;
+    background-position: top center;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 40px 0 0;
+  }
+
+  &__code {
+    margin-top: 4px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-text-color;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 166px;
+    height: 16px;
+    margin: 0;
+    margin-top: 16px;
+  }
+
+  &__reviews {
+    font-weight: 400;
+    font-size: 10px;
+    text-align: right;
+  }
+
+  &__price {
+    margin: 0;
+    margin-top: 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__price-title {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $secondary-text-color;
+  }
+
+  &__price-amount {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $main-text-color;
+    text-align: right;
+  }
```

No need to duplicate selectors
```suggestion
  &__price {
    margin: 0;
    margin-top: 24px;
    display: flex;
    justify-content: space-between;
    align-items: center;
    
    &-title {}
    
    &-amount {}
  }
```